packages/dso-toolkit/gulp/build-svg-spritesheet.js

```diff
@@ -13,135 +13,131 @@ import { sassTransformer } from "./transformers/sass.transformer.js";
+                  .map((v) => ({ selector: `${id}-${v.selector}`, color: v.color })),
               ];
 
-              const iconIds = variants.map((v) => v.selector);
-
-              symbol.before(`<!-- START: ${iconIds.join(", ")} -->`).after(`<!-- END: ${iconIds.join(", ")} -->`);
-
               variants.forEach((variant, index) => {
-                const x = (position + index) * (canvas + gutter) + gutter / 2;
-                const svg = symbol.clone().removeAttr("id");
-                $(svg).each((_index, element) => (element.tagName = "svg"));
-
-                if (variant.color) {
-                  svg.find('[fill="currentColor"]').attr("fill", variant.color);
-                }
+                const x = (position + index) * (canvasSize + gutterSize) + gutterSize / 2;
 
                 const view = $("<view>")
-                  .attr("id", `img-${variant.selector}`)
-                  .attr("viewBox", [x, 0, canvas, canvas].join(" "));
+                  .attr("id", `${variant.selector}`)
+                  .attr("viewBox", [x, 0, canvasSize, canvasSize].join(" "));
+
+                const svgElement = $("<svg>").attr("id", variant.selector).attr("viewBox", $(symbol).attr("viewBox"));
 
-                symbol.before(view);
+                $(symbol)
+                  .children()
+                  .clone()
+                  .each((_, child) => {
+                    if (variant.color && $(child).attr("fill") === "currentColor") {
+                      $(child).attr("fill", variant.color);
+                    }
+                    svgElement.append(child);
+                  });
 
-                const g = $("<g>").attr("transform", `translate(${x})`).append(svg);
+                const group = $("<g>").attr("transform", `translate(${x})`).append(svgElement);
 
-                symbol.before(g);
+                $(symbol).before(view).before(group);
               });
 
-              return position + variants.length;
-            }, 0);
-
-            $(":root").attr(
-              "viewBox",
-              [
-                (positions * (canvas + gutter)) / 2 - (canvas + gutter) / 2 + gutter / 2,
-                0,
-                positions * (canvas + gutter) + gutter,
-                canvas,
-              ].join(" "),
-            );
-          },
-          parserOptions: { xmlMode: true },
-        }),
-      )
-      .pipe(prettier({ plugins: ["@prettier/plugin-xml"] }))
-      .pipe(rename("dso-icons.svg"))
-      .pipe(gulp.dest(distPath))
-      .on("end", resolve)
-      .on("error", reject);
-  });
+              $(symbol).remove();
+              position += variants.length;
+            }
+          });
+        },
+        parserOptions: { xmlMode: true },
+      }),
+    )
+    .pipe(rename(iconsWithStyles ? outPutNameIconsWithVariants : outPutNameIcons))
+    .pipe(gulp.dest(distPath));
+}
+
+export async function buildSvgSpritesheet() {
```

Comment:
In deze functie worden er meerdere spritesheets gegenereerd:

```suggestion
export async function buildSvgSpritesheets() {
```

Reply:
done